css/main.scss

```diff
@@ -1857,11 +1855,11 @@ fieldset {
   }
 }
 textarea {
-  width: calc(100% - 16px);
+  width: 100%;
```

Comment:
came about this css. It is unclear that "textarea" is only affecting API explorer. Can you change to class name specific to API explorer? cc @hubert-deriv